peerscout_api/main.py

```diff
@@ -99,6 +92,24 @@
     WHERE person.person_id IN UNNEST({person_ids})
```

Comment:
We probably should be using [named parameters](https://cloud.google.com/bigquery/docs/parameterized-queries#python) for the `person_ids`. Otherwise we could be vulnerable to SQL injection.

The `project` and `dataset` parameter are a bit different because we configure them.
But the `person_ids` are coming from other sources, including from the API request.